lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -309,6 +311,18 @@ class << self
           alias_method :split_request?, :split_request_size_check?
         end
       end
+
+      version_arr = Elasticsearch::Transport::VERSION.split('.')
+
+      if (version_arr[0].to_i < 7) || (version_arr[0].to_i == 7 && version_arr[1].to_i < 2)
```

Comment:
Can we use `Elasticsearch::Transport::VERSION < "7.2.0"` here?

Reply:
Nope
When Transport version will be 10.0.0 it won't work, we'll get an error

Reply:
Oh, I've missed the pitfall. Thanks. :smile: